park-dashboard/src/Components/Searchbar/Searchbar.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from "react";
+import Select from "react-select";
+import axios from "axios";
+import { Link } from "react-router-dom";
+import "./searchbar.css"
+
+const BASE_URL = "https://developer.nps.gov/api/v1";
+const API_KEY = "OaR7jmqSa22JAcsym9lVfStp58LmCqH9JdZUPEH7";
+
+const Searchbar = () => {
+  const [parkName, setParkName] = useState([]);
+
+  useEffect(() => {
+    const fetchNames = async () => {
+      try {
+        const response = await axios.get(`${BASE_URL}/parks`, {
```

Comment:
Can you look into @rutujapadgilwar's `LoadParkData` class as it's also accessing the same API and caching it to Local storage. Maybe can you check to see if we can reuse that here to reduce the number of calls we make to the server.

Reply:
I am not able to fetch data from LoadParkData since it is showing me an empty array. 
But as per your suggestion, I have removed the API call from here and now data is passed from my previously fetched API call from StatesData which is storing data in cache in first call.